File: trail1/정수 사이의 배수/multiplication-between-numbers.py
lst = list(map(int, input().split()))
A, B = lst[0], lst[1]
ssum = 0
cnt = 0
for i in range(A, B+1):
    if not i % 5 or not i % 7:
        ssum += i
        cnt += 1
print(f"{ssum} {ssum/cnt:.1f}")

File: trail1/정수 사이의 배수/multiplication-between-numbers.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        int ssum = 0, cnt = 0;
        var(A, B) = (int.Parse(arr[0]), int.Parse(arr[1]));
        for (int i=A; i<=B; i++) {
            if (i % 5 == 0 || i % 7 == 0) {
                ssum += i;
                cnt += 1;
            }
        }
        Console.WriteLine($"{ssum} {(double)ssum/cnt:F1}");
    }
}
